### PR TITLE
normalize license expire date to prevent outstanding value on Java 12

### PR DESCRIPTION
**Pull request by ibmibmibm**
Date(Long.MAX_VALUE) is 292278994-08-17T07:12:55.807Z on Java 12, and
make [Ultrasonic](https://github.com/ultrasonic/ultrasonic) failed to parse.

Signed-off-by: Shen-Ta Hsieh <ibmibmibm.tw@gmail.com>

### CODE REVIEW

File: airsonic-main/src/main/java/org/airsonic/player/controller/SubsonicRESTController.java [hunk 1]
```diff
@@ -170,9 +170,10 @@ public void getLicense(HttpServletRequest request, HttpServletResponse response)
 
         license.setEmail("airsonic@github.com");
         license.setValid(true);
-        Date neverExpireDate = new Date(Long.MAX_VALUE);
-        license.setLicenseExpires(jaxbWriter.convertDate(neverExpireDate));
-        license.setTrialExpires(jaxbWriter.convertDate(neverExpireDate));
+        Date nextCentenary = new Date();
```
Comment (jvoisin) [inline airsonic-main/src/main/java/org/airsonic/player/controller/SubsonicRESTController.java]:
```suggestion
        Date nextCentury = new Date();
        nextCentury.setYear(nextCentury.getYear() + 100);
        license.setLicenseExpires(jaxbWriter.convertDate(nextCentury));
        license.setTrialExpires(jaxbWriter.convertDate(nextCentury));
```

**Comment by jvoisin** [comment]
I fixed the current master, please rebase on it and I'll be happy to merge once everything is green :)

File: airsonic-main/src/main/java/org/airsonic/player/controller/SubsonicRESTController.java (hunk 1 above)
Comment (ibmibmibm) [reply airsonic-main/src/main/java/org/airsonic/player/controller/SubsonicRESTController.java]: changed to `farFuture`.

**Comment by eharris** [comment]
Thanks.  Hopefully this will be merged soon.

**Pull request merged**